src/DebugSql.php

```diff
@@ -77,8 +77,8 @@ public static function sql(Query $query, $showValues = true, $showHtml = null, $
         }
 
         $trace = Debugger::trace(['start' => 1, 'depth' => $stackDepth + 2, 'format' => 'array']);
-        $file = $trace[$stackDepth]['file'];
-        $line = $trace[$stackDepth]['line'];
+        $file = isset($trace[$stackDepth]) ? $trace[$stackDepth]['file'] : 'n/a';
```

Comment:
You might want to check that there is a file/line key set. Not all points in a stack trace will have file/line sadly.

Reply:
done.